chromadb/api/segment.py

```diff
@@ -319,7 +319,7 @@ def delete_collection(
                 existing[0].id, tenant=tenant, database=database
             )
             for s in self._manager.delete_segments(existing[0].id):
-                self._sysdb.delete_segment(s)
+                self._sysdb.delete_segment(existing[0].id, s)
```

Comment:
note to self - when we address transactionality the deleting of segments really should not be here